Client/Frontend/Fakespot/Views/FakespotHighlightGroupView.swift

```diff
@@ -0,0 +1,139 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import ComponentLibrary
+import UIKit
+
+struct FakespotHighlightGroupViewModel {
+    let highlightGroup: HighlightGroup
+}
+
+class FakespotHighlightGroupView: UIView, ThemeApplicable, Notifiable {
+    private struct UX {
+        static let horizontalSpace: CGFloat = 8
+        static let verticalSpace: CGFloat = 8
+        static let imageSize: CGFloat = 24
+        static let imageMaxSize: CGFloat = 58
+        static let titleFontSize: CGFloat = 15
+        static let highlightFontSize: CGFloat = 13
+    }
+
+    private lazy var itemImageView: UIImageView = .build()
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .headline,
+            size: UX.titleFontSize,
+            weight: .semibold)
+        label.numberOfLines = 0
+    }
+
+    private lazy var highlightLabel: FakespotFadeLabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .body,
+            size: UX.highlightFontSize)
+        label.numberOfLines = 0
+        label.lineBreakMode = .byClipping
+    }
+
+    private var highlightLabelLeadingConstraint: NSLayoutConstraint?
+    private var imageHeightConstraint: NSLayoutConstraint?
+
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
```

Comment:
In general, the default value be in the init, for testing purposes.

Reply:
True, but then we'd need to have a custom init which means we can't use `.build()` anymore.